client/client-multi/client-c/src/kaa/platform-impl/common/key_utils.c

```diff
@@ -34,8 +34,8 @@
 
 #include <kaa_rsa_key_gen.h>
 
-#define KAA_PRIVATE_KEY_STORAGE "private.key"
-#define KAA_PUBLIC_KEY_STORAGE "public.key"
+#define KAA_PRIVATE_KEY_STORAGE "key.private"
+#define KAA_PUBLIC_KEY_STORAGE "key.public"
```

Comment:
SSH stores key pair as `key_name` for private and `key_name.pub` for public. It might be wise to use same naming scheme.